problems/4.arrays.js

```diff
@@ -2,8 +2,12 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+const firstAndLastAverage = (arr) => {
+  if (arr.length < 2) {
+    return null
+  } else {
```

Comment:
Technically don't need else. 
The if will return, all that's left is if we didn't return